packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -268,6 +268,156 @@ function extractProxyAccessorsFor(
   ];
 }
 
+/**
+ * Prepend expressions to the field initializer. If the initializer is not defined,
+ * this function will add a trailing `void 0` expression
+ *
+ * @param {t.Expression[]} expressions
+ * @param {(NodePath<
+ *     t.ClassProperty | t.ClassPrivateProperty | t.ClassAccessorProperty
+ *   >)} fieldPath
+ */
+function prependExpressionsToFieldInitializer(
+  expressions: t.Expression[],
+  fieldPath: NodePath<
+    t.ClassProperty | t.ClassPrivateProperty | t.ClassAccessorProperty
+  >,
+) {
+  const initializer = fieldPath.get("value");
+  if (initializer.node) {
+    expressions.push(initializer.node);
+  } else {
+    expressions.push(t.unaryExpression("void", t.numericLiteral(0)));
```

Comment:
We can probably get rid of the trailing `void 0` here since `_protoInit()` returns `undefined`.